pom.xml

```diff
@@ -1058,6 +1081,86 @@
         <artifactId>structure-maven-plugin</artifactId>
         <version>1.0</version>
       </plugin>
+      <plugin>
+        <groupId>com.mycila</groupId>
+        <artifactId>license-maven-plugin</artifactId>
+        <version>4.3</version>
+        <configuration>
+          <properties>
+            <owner>The OpenNMS Group, Inc.</owner>
+            <email>license@opennms.com</email>
+          </properties>
+          <licenseSets>
+            <licenseSet>
+              <header>src/main/resources/license-header.txt</header>
+              <!-- some of these excludes are aspirational, for when we expand this beyond the basics 😅 -->
+              <excludes>
+                <!-- short term, let's exclude everything but c/c++/groovy/java/js/ts -->
+                <exclude>**/*.css</exclude>
```

Comment:
Should we also exclude `.tsx` files?

Reply:
I mean... we don't have any?  :)

```
~/git/opennms-develop $ find * -name \*.tsx
~/git/opennms-develop $
```

And ultimately I'd like to add the other formats too, there just needs to be some work on excluding some things that don't need anything added.